### PR TITLE
Artsiom/develop

### PR DESCRIPTION
**Pull request by ArteGon**
Add snackbar for our App
Fix bugs with create and update events
Begin develop with filters 
Also implementation by Dima

### CODE REVIEW

File: src/components/events/events.tsx [hunk 1]
```diff
@@ -20,35 +21,76 @@ const useStyles = makeStyles(() => {
 });
 
 interface IEvent {
-  id?: string;
+  id?: string | number;
 }
 
 const Events: FunctionComponent = () => {
   const classes = useStyles();
+  const [loadingData, setLoadingData] = useState<Boolean>(true);
```
Comment (vchepurnaya) [inline src/components/events/events.tsx]:
```suggestion
  const [loadingData, setLoadingData] = useState<boolean>(true);
```

File: src/components/events/events.tsx [hunk 2]
```diff
@@ -20,35 +21,76 @@ const useStyles = makeStyles(() => {
...
+
+  const [openSnackbar, setopenSnackbar] = useState<boolean>(false);
+  const [alertSeverity, setAlertSeverity] = useState<'success' | 'error' | 'warning' | 'info' | undefined>(undefined);
```
Comment (vchepurnaya) [inline src/components/events/events.tsx:32]: i think it's better to fix this type

**Review by DmitryPateew — DISMISSED** [review]

**Review by ZmicerBoksha — DISMISSED** [review]

**Review by vchepurnaya — DISMISSED** [review]

**Review by vchepurnaya — DISMISSED** [review]

File: src/api/api.ts [hunk 3]
```diff
@@ -47,23 +52,24 @@ export const eventsApi = {
...
+        // format: formData.format,
+        // country: formData?.country,
+        // city: formData?.city,
+        ...formData,
         image: 1,
         creatorEvent: 1,
         employee: 1,
         eventType: 1,
       })
       .then(response => {
```
Comment (ZmicerBoksha) [inline src/api/api.ts]: .then(response => response);

File: src/api/api.ts [hunk 4]
```diff
@@ -29,8 +29,13 @@ const instance = axios.create({
...
+  getEvents(searchParam?: string) {
+    let urlForRequest = '/event/all';
+    if (searchParam) {
+      urlForRequest += `?search=${searchParam}`;
+    }
+
+    return instance.get(urlForRequest).then(response => {
       console.log(response);
       return response.data;
     });
```
Comment (ZmicerBoksha) [inline src/api/api.ts]: .then(response => response);

File: src/api/api.ts [hunk 5]
```diff
@@ -72,28 +78,29 @@ export const eventsApi = {
...
+        // format: formData.format,
+        // country: formData?.country,
+        // city: formData?.city,
+        ...formData,
         image: 1,
         creatorEvent: 1,
         employee: 1,
         eventType: 1,
       })
       .then(response => {
```
Comment (ZmicerBoksha) [inline src/api/api.ts]: .then(response => response);

File: src/api/api.ts [hunk 6]
```diff
@@ -72,28 +78,29 @@ export const eventsApi = {
...
-        console.log(response);
+        return response;
       });
   },
   deleteEvent(eventId: number) {
     return instance.delete(`/event/${eventId}`).then(response => {
```
Comment (ZmicerBoksha) [inline src/api/api.ts]: .then(response => response);

**Review by vchepurnaya — APPROVED** [review]

**Review by ZmicerBoksha — APPROVED** [review]

**Review by DmitryPateew — APPROVED** [review]

**Pull request merged**